setup.py

```diff
@@ -35,7 +35,7 @@
         'Programming Language :: Python :: 3.6',
     ],
 
-    install_requires=['jsonpickle', 'wrapt', 'requests'],
+    install_requires=['jsonpickle', 'wrapt', 'requests', 'future'],
```

Comment:
Do you need future if you remove all print statement?

Reply:
No future will still be needed for the super() builtin
from builtins import super